README.md

```diff
@@ -60,27 +60,49 @@ For first-time jspm users, you will need to configure it to use GitHub:
 
 Follow the steps to enable GitHub access.
 
-### Getting started
+### Getting started: docker development
+Clone this repository.
+
+Create the file `/etc/resolver/dev` with the following contents:
```

Comment:
Yup, it's an absolute must to get DNS resolving between the host and the container working on OS X, and it does need root permissions to accomplish (similar to editing /etc/hosts)